src/models/github-server-app.ts

```diff
@@ -130,18 +169,27 @@ GitHubServerApp.init({
 		allowNull: false
 	},
 	secrets: encrypted.vault("secrets"),
```

Comment:
Leave the removal of this field alone for now. Will need to consider prod db (both schema and data) migration properly in another PR later.